### DISCUSSION TITLE
Gather data from about section

### DISCUSSION THREAD

**Discussion by nikunj113** [omkarcloud/google-maps-scraper#169] [Ideas]
Hi! 

First of great tool!! Extremely useful.  I've been using it to get leads on clients and this has made my life so much easier.  There is just one thing I was hoping the tool had.  I've been looking to extract data on which businesses accept credit card as payments however that is stored in the about section under payments of google maps search.  Is there a way to extract this data?

**Comment by Chetan11-dev** [comment]
about section is extracted
